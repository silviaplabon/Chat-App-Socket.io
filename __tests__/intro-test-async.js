import React from 'react';
import renderer from 'react-test-renderer';
import MyScreenAsync from '../src/practice/MyScreenAsync';

const tree = renderer.create(< MyScreenAsync/>)
// jest.runAllTimers();
test('snapshot',()=>{
    expect(tree).toMatchSnapshot();
})
test('call timeout',()=>{
    renderer.act(()=>jest.runAllTimers())
    const text=tree.root.findByProps({testID:'myText'}).props;
    expect(text.children).toEqual('timeout is called');
})