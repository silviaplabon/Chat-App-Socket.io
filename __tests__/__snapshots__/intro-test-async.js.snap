// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    testID="myText"
  >
    
  </Text>
</View>
`;
